test/UserService.Mappers.UnitTests/Requests/Certificate/DbUserCertificateMapperTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using LT.DigitalOffice.UnitTestKernel;
+using LT.DigitalOffice.UserService.Mappers.Db;
+using LT.DigitalOffice.UserService.Models.Db;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.User.Certificates;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.UserService.Mappers.UnitTests.Requests.Certificate
```

Comment:
better use namespace LT.DigitalOffice.UserService.Mappers.Db.UnitTests

Reply:
Why?

Reply:
because you can better not use namespace ` LT.DigitalOffice.UserService.Mappers.Db`

Reply:
fixed